Week-3/Session9/JavaScript arrays/script.js

```diff
@@ -0,0 +1,25 @@
+// Put your solution here
+function divideArrays(nums) {
+    let evenNums=[],oddNums=[];
+    for(let i=0 ; i<nums.length ; i++){
+        if(nums[i] % 2 === 0){
+            evenNums.push(nums[i])
+        } else {
+          oddNums.push(nums[i])  
+        }
+    }
+    evenNums.sort();
+    oddNums.sort();
+    console.log("Even Numbers:");
+    if(evenNums.length == 0){
```

Comment:
[suggestion]If you aware of chaining operator, instead of if-else loop you can use like this
1) console.log(evenNums?.length)
2) evenNums && evenNums.length

Reply:
Same for the oddNums

Reply:
This is okay for now, but just adding my suggestion's

Reply:
Sure Kunal will try to follow the same. Thanks.